src/github.com/matrix-org/dendrite/clientapi/routing/createroom.go

```diff
@@ -87,13 +87,15 @@ func (r createRoomRequest) Validate() *util.JSONResponse {
 			}
 		}
 	}
-	switch r.Preset {
-	case presetPrivateChat, presetTrustedPrivateChat, presetPublicChat:
```

Comment:
Why not just add `case "": break`?
Would make the diff shorter :smirk:

Reply:
Oh, duh. And go doesn't even need `break`. Super short!